File: fix.py
import re
with open('src/pages/AdminHubPage.tsx', 'r') as f:
    code = f.read()

code = code.replace('headers.join(",") + "\\\n"', 'headers.join(",") + "\\\\n"')
code = code.replace('.join("\\\n");', '.join("\\\\n");')
with open('src/pages/AdminHubPage.tsx', 'w') as f:
    f.write(code)

File: fix.js
const fs = require('fs');
let code = fs.readFileSync('src/pages/AdminHubPage.tsx', 'utf8');
code = code.replace(/headers\.join\(\",\"\)\s*\+\s*\"\n\"/g, 'headers.join(",") + "\\n"');
code = code.replace(/\.join\(\"\n\"\);/g, '.join("\\n");');
fs.writeFileSync('src/pages/AdminHubPage.tsx', code);
